language/diem-framework/modules/NetworkIdentity.move

```diff
@@ -0,0 +1,91 @@
+address 0x1 {
+/// Module managing Diemnet NetworkIdentity
+module NetworkIdentity {
+    use 0x1::DiemTimestamp;
+    use 0x1::Event::{Self, EventHandle};
+    use 0x1::Signer;
+    use 0x1::Vector;
+
+    /// An updatable `address` list with update notifications
+    struct AccountList has key {
+        accounts: vector<address>,
+        account_change_events: EventHandle<AccountListChangeNotification>
+    }
+
+    /// Message sent when there are updates to the `AccountList`.
+    struct AccountListChangeNotification has drop, store {
+        /// The new accounts
+        new_accounts: vector<address>,
```

Comment:
A couple questions about this field, and if there are possibly some things we should enforce on it:
1. Should this be a set?
2. Is there an upper bound that we can place on the number of addresses allowed here? e.g., no more than 256 addresses? If so we should probably do that. This is especially important as anyone can publish this and add addresses to it, so someone could make this grow quite large...

Reply:
Yes, it should be a set.  There could be an upper limit on the number of accounts, but if it got to a point where the size needed to be larger, we'd need two resources to keep track of it.